ducktape/tests/reporter.py

```diff
@@ -304,3 +307,41 @@ def format_report(self):
 
     def report(self):
         self.format_report()
+
+
+class TestSuiteGeneratorReporter(SummaryReporter):
+
+    def __init__(self, results):
+        super().__init__(results)
+        self.separator = ''.join(["=" * self.width])
+
+    def format_line(self, result):
+        line = f'{result.file_name}::{result.cls_name}.{result.function_name}'
+        if result.injected_args:
+            injected_args_str = json.dumps(result.injected_args, separators=(',', ':'))
+            line += f'@{injected_args_str}'
+        return line
+
+    def dump_test_suite(self, lines):
+        print(self.separator)
+        print('FAILED TEST SUITE')
+        suite = {self.results.session_context.session_id: lines}
+        file_path = Path(self.results.session_context.results_dir) / "rerun-failed.yml"
```

Comment:
can you make the file an optional argument?

Reply:
do you mean the name of the file? I don't think there's any point, and we don't do it for other files (like report.html). This is stored in the results folder, so unique per test run